install_config/configuring_openstack.adoc

```diff
@@ -40,7 +39,55 @@ Consult your OpenStack administrators for values of the `*OS_*` variables, which
 are commonly used in OpenStack configuration.
 
 [[openstack-configuring-masters]]
-== Configuring Masters
+== Configuring OpenStack for {product-title} Masters
```

Comment:
This structure makes sense (to me) for the AWS and OpenStack topics, because they have the separate sections about manually modifying the master and node configs. I'm leaving it as is, and I'll see what the rest of the docs team thinks.